aptos-move/aptos-gas-schedule/src/ver.rs

```diff
@@ -9,7 +9,8 @@
 ///
 /// Change log:
 /// - V11
-//    - Ristretto255 natives (point cloning & double-scalar multiplication) and Bulletproofs natives
+///   - Making resource group charge on first read independent of BTreeMap serialization.
+///   - Ristretto255 natives (point cloning & double-scalar multiplication) and Bulletproofs natives
```

Comment:
hmm, is V11 pending release or not? it's probably safer to bump to V12

Reply:
@msmouse ? :D

Reply:
https://github.com/aptos-labs/aptos-core/blob/aptos-node-v1.6.2/aptos-move/aptos-gas/src/gas_meter.rs#L67

10 is the latest for 1.6.2